latest/PIT/pit_basic.yaml

```diff
@@ -0,0 +1,132 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/PIT/pit_basic-1.0.0
+
+title: PIT-specific common metadata
+description: |
+  This schema contains PIT-specific common metadata, excluding metadata
+  that can typically be found in a Roman mission science product schema.
+type: object
+properties:
+  community_product_id:
+    title: Project Infrastructure Team name
+    description: |
+      The short name of the Roman Project Infrastructure Team (PIT)
+      who created this product.
+    type: string
+    maxLength: 5
+    enum: ["GRS", "HLIS", "RAPID", "RGES", "SN"]
+  community_product_investigator:
+    title: PIT Principal Investigator
+    description: |
+      The name of the PIT's Principal Investigator.
+    type: string
+  community_product_archive_lead:
+    # Optional
+    title: PIT staff lead for MAST ingest
+    description: |
+      The name(s) of PIT staff members who led coordination with MAST
+      on archival ingest.
+    type: string
+  doi:
+    title: PIT Digital Object Identifier
+    description: |
+      The Digital Object Identifier (DOI) for this PIT collection's
+      presence in MAST.
+    type: string
+  file_version:
+    title: Version of this file
+    description: |
+      Version of this file, to be incremented with any delivery
+      that replaces this file. Usually not tied to data release
+      identifiers or software version. Should match version field
+      of the file name.
+    type: string
+    maxLength: 9
+  data_release:
```

Comment:
The SOC schemas use `data_release_id`:
https://github.com/spacetelescope/rad/blob/9fda7555b39693cc6264b007c832b4a9c6be241f/latest/meta/l3_common.yaml#L14
Is this referring to essentially the same thing and does it make sense to use the same name?

Reply:
Thanks for the clarification. Not reusing `data_release_id` sounds good to me given the differences. `pit_data_release_id` sounds pretty intuitive.